### DISCUSSION TITLE
about Self Duration

### DISCUSSION THREAD

**Discussion by dongguabai** [apache/skywalking#7995] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/31611757/138602939-73df4d02-57cf-4fd2-b3ee-50436fbdbd0a.png)
Hello, I met a question today, the total duration is 8710ms，but the self duration is 6504ms.In the official document, I didn't find a introduction to Self Duration. How do I understand the meaning of Self Duration?

**Comment by wu-sheng** [comment, marked as answer]
The duration of the span excluding all children spans is the self-duration.
